package.json

```diff
@@ -12,7 +12,7 @@
     "build:bundle": "rollup -c ./config/rollup.config.js",
     "build:type": "tsc --project ./config/tsconfig.type.json",
     "lint": "eslint './src/**/*.{ts,tsx}'",
-    "prettier": "prettier --check ./**/*.{ts,tsx,js,json}"
+    "prettier": "prettier --check ./**/*.{ts,tsx,js,json} ./**.js"
```

Comment:
Why do you add another for JavaScript files here?

Reply:
do prettier check babel.config.js, .eslintrc.js ... etc